engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -117,48 +115,14 @@ class WorkflowManagerActor(params: WorkflowManagerActorParams)
   private val tag = self.path.name
 
   private var abortingWorkflowToReplyTo = Map.empty[WorkflowId, ActorRef]
-  private var shutdownHookThreadOption: Option[ShutdownHookThread] = None
+  private var nextPollCancellable: Option[Cancellable] = None
+  private var preventNewWorkflowsFromStarting: Boolean = false
```

Comment:
I was figuring the last part was the reason :)

It did seem like it'd be a waste to do that but OTOH I'm afraid of setting up a slippery slope where we start having states and pseudo-states mixed all over the place. I'm sure you can find examples of us doing this already as well as examples of lots of copypasta due to keeping it clean.